manager/middlewares/rbac.go

```diff
@@ -59,54 +50,56 @@ func makeClient(identity string) *api.Client {
 	return &client
 }
 
+// nolint: goconst
+// for short lists like that is slice.Contains() faster than map lookup _, ok := map[key]
+func expandedPermission(perm string) []string {
+	comp := strings.SplitAfterN(perm, ":", 3)
+	expandedPerm := []string{perm,
+		comp[0] + comp[1] + "*",
+		comp[0] + "*:" + comp[2],
+		"*:" + comp[1] + comp[2],
+		comp[0] + "*:*",
+		"*:" + comp[1] + "*:*",
```

Comment:
I think this should be
```suggestion
		"*:" + comp[1] + "*",
```